index.html

```diff
@@ -121,39 +146,38 @@
        <div class="feedback-popup">
          <p>Напишите нам</p>
          <button type="button" class="cross"></button>
-         <form action="#">
+         <form action="https://echo.htmlacademy.ru" method="post" id="feedback-form">
           <div class="name-email clearfix">
             <div class="name-field">
               <label for="your-name">Ваше имя:</label>
-              <input type="text" id="your-name" name="feedback" class="feedback-name" placeholder="Представьтесь, пожалуйста">
+              <input type="text" id="your-name" name="feedback" class="feedback-name" placeholder="Представьтесь, пожалуйста" >
             </div>
             <div class="email-field">
               <label for="e-mail">Ваш e-mail:</label>
```

Comment:
Input's id was changed to "your-email", so label's "for" attribute has to be changed too.